arlas-core/src/main/java/io/arlas/server/core/services/ExploreService.java

```diff
@@ -333,7 +357,7 @@ public AggregationResponse aggregate(MixedRequest request,
         FluidSearchService fluidSearch = getFluidSearch(collectionReference);
         applyFilter(collectionReference.params.filter, fluidSearch);
```

Comment:
Should we externalize also the collectionFilter ? Maybe in another PR ?

Reply:
I dont think so, the collection filter is like a filter from the ui app, a classic combination of AND of OR

Reply:
Ok then LGTM !